testdata/go/v3/memcached-operator/Makefile

```diff
@@ -88,7 +88,7 @@ vet: ## Run go vet against code.
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: manifests generate fmt vet ## Run tests.
```

Comment:
looks like you may have some stale changes, try rebasing on the upstream master

Reply:
I did the rebase still it came in the change.

Reply:
`make generate` solved this.